src/impl_methods.rs

```diff
@@ -196,6 +197,26 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
         debug_assert!(self.pointer_is_inbounds());
     }
 
+    pub fn select_along(&self, axis: Axis, indices: &[Ix]) -> OwnedArray<A, D>
```

Comment:
I think select is a good keyword (probably got it from daniel's issue). Do you have a suggestion for an even better method name here? I was thinking that this is maybe only one kind of such a method, this is the along an axis selection. `select_along` would suggest "along an axis" maybe `select_axis` is better? Only `select`? Something else.

This method is actually in the wrong place. It's not so important that it should be before .get() in the docs. I guess it's not a perfectly sorted order. Below the subview methods seems ok.